lib/specific_todo.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+class SpecificTodo extends StatelessWidget {
+  final String todo;
+  final bool isDone;
+  final int index;
+
+  const SpecificTodo(
+      {super.key,
+      required this.todo,
+      required this.isDone,
+      required this.index});
```

Comment:
formatting nit: use trailing commas

```suggestion
  const SpecificTodo({
    super.key,
    required this.todo,
    required this.isDone,
    required this.index,
  });
```